getputremovetest/src/main/java/org/infinispan/UtfBenchmark.java

```diff
@@ -1,14 +1,30 @@
 package org.infinispan;
 
 import java.io.IOException;
+import java.util.concurrent.TimeUnit;
 
 import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.CompilerControl;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.infra.Blackhole;
 
+@Fork(value=1, jvmArgs = {
+      "-Xmx10G",
```

Comment:
10G seems quite excessive for this test...

Reply:
I'll just remove `@Fork` as it can all be specified via the CLI.